lib/menu.rb

```diff
@@ -0,0 +1,9 @@
+class Menu
+  
+  attr_accessor :dish, :price
+
+  def initialize(dish,price = 1)
+    @dish = dish
+    @price = price
+  end
```

Comment:
Nice short class, succinct with the Menu

Reply:
Thanks, Chuka! Now I'm looking at your Takeaway Class & menu set up, I think I need to work on this further, perhaps with a new Class, so thank you!